core/src/main/java/org/apache/gravitino/authorization/PermissionManager.java

```diff
@@ -116,16 +117,23 @@ User grantRolesToUser(String metalake, List<String> roles, String user) {
               });
 
       Set<String> catalogs = Sets.newHashSet();
```

Comment:
I think you can move this line code
```
Set<String> catalogs = Sets.newHashSet();
```
into `callAuthorizationPluginForSecurableObjects()` body, because you didn't use this `catalogs` variable in another place.

Reply:
OK, I moved it.